BitwardenShared/UI/Vault/Vault/VaultList/VaultListProcessor.swift

```diff
@@ -241,6 +242,12 @@ extension VaultListProcessor {
         state.canShowVaultFilter = await services.vaultRepository.canShowVaultFilter()
     }
 
+    /// Checks available item types user can create.
+    ///
+    private func checkItemTypesUserCanCreate() async {
```

Comment:
⛏️ Same as the other one IMO this is "loading" instead of "checking" item types.